API/src/main/java/io/github/jwdeveloper/tiktok/data/events/TikTokLinkMicArmiesEvent.java

```diff
@@ -54,4 +53,12 @@ public TikTokLinkMicArmiesEvent(WebcastLinkMicArmies msg) {
         picture = Picture.map(msg.getImage());
         battleStatus = msg.getBattleStatus();
     }
-}
+
+    /**
+     battleStatus of 1 is Ongoing battle & 2 is Finished Battle
+     @return true if battle is finished otherwise false
+     */
```

Comment:
Please create Enum for this. Also you can create Enum in proto. 
enum BattleStatus
{
  ongoing =1
  finished = 2
}

Reply:
Fixed